### PR TITLE
fix(token): add mIO and IO token classes to exports

### PR DESCRIPTION
**Pull request by dtfiedler**
These are helper classes to easily convert between mIO and IO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -378,7 +383,7 @@ const gateway = await arIO.getGateway({
 {
   "end": 0,
   "observerWallet": "IPdwa3Mb_9pDD8c2IaJx6aad51Ss-_TfStVwBuhtXMs",
-  "operatorStake": 250000,
+  "operatorStake": 250000, // value in mIO
```
Comment (kunstmusik) [inline README.md:386]: Should maybe update this readme example as I think this wouldn't be a valid amount of mIO for operatorStake

File: README.md [hunk 2]
```diff
@@ -427,7 +432,7 @@ const gateways = await arIO.getGateways();
   "QGWqtJdLLgm2ehFWiiPzMaoFLD50CnGuzZIPEdoDRGQ": {
     "end": 0,
     "observerWallet": "IPdwa3Mb_9pDD8c2IaJx6aad51Ss-_TfStVwBuhtXMs",
-    "operatorStake": 250000,
+    "operatorStake": 250000, // value in mIO
```
Comment (kunstmusik) [inline README.md:435]: Should maybe update this readme example as I think this wouldn't be a valid amount of mIO for operatorStake

File: README.md [hunk 3]
```diff
@@ -507,7 +512,7 @@ const records = await arIO.getArNSRecords();
   },
   "ar-io": {
     "contractTxId": "eNey-H9RB9uCdoJUvPULb35qhZVXZcEXv8xds4aHhkQ",
-    "purchasePrice": 17386.717520731843,
+    "purchasePrice": 17386.717520731843, // value in mIO
```
Comment (kunstmusik) [inline README.md:515]: Should maybe update this readme example as I think this wouldn't be a typical amount of mIO for purchasePrice

File: README.md [hunk 4]
```diff
@@ -640,7 +645,7 @@ const observers = await arIO.getPrescribedObservers();
   {
     "gatewayAddress": "BpQlyhREz4lNGS-y3rSS1WxADfxPpAuing9Lgfdrj2U",
     "observerAddress": "2Fk8lCmDegPg6jjprl57-UCpKmNgYiKwyhkU4vMNDnE",
-    "stake": 10000,
+    "stake": 10000, // value in mIO
```
Comment (kunstmusik) [inline README.md:648]: Should maybe update this readme example as I think this wouldn't be a valid amount of mIO for operatorStake

File: README.md [hunk 5]
```diff
@@ -673,7 +678,7 @@ const previousEpochObservers = await arIO.getPrescribedObservers({
   {
     "gatewayAddress": "2Ic0ZIpt85tjiVRaD_qoTSo9jgT7w0rbf4puSTRidcU",
     "observerAddress": "2Ic0ZIpt85tjiVRaD_qoTSo9jgT7w0rbf4puSTRidcU",
-    "stake": 10000,
+    "stake": 10000, // vault in mIO
```
Comment (kunstmusik) [inline README.md:681]: Should maybe update this readme example as I think this wouldn't be a typical amount of mIO for operatorStake

File: src/constants.ts [hunk 6]
```diff
@@ -30,3 +30,5 @@ export const ARNS_TESTNET_REGISTRY_TX =
 
 export const ARNS_DEVNET_REGISTRY_TX =
   '_NctcA2sRy1-J4OmIQZbYFPM17piNcbdBPH2ncX2RL8';
+
+export const MIO_PER_IO = 1000000;
```
Comment (kunstmusik) [inline src/constants.ts:34]: Could we change this to 1_000_000 to make it a little easier to read?

Comment (dtfiedler) [reply src/constants.ts]: [7ff3530](https://github.com/ar-io/ar-io-sdk/pull/85/commits/7ff35305c978f2c9defb01bb4e5fd878a7bc9813)

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**